huaweicloud/services/drs/resource_huaweicloud_drs_job.go

```diff
@@ -467,31 +469,51 @@ func resourceJobUpdate(ctx context.Context, d *schema.ResourceData, meta interfa
 	if err != nil {
 		return diag.Errorf("error creating DRS v3 client, error: %s", err)
 	}
-
-	detailResp, err := jobs.Get(client, jobs.QueryJobReq{Jobs: []string{d.Id()}})
+	clientV5, err := conf.DrsV5Client(region)
 	if err != nil {
-		return common.CheckDeletedDiag(d, parseDrsJobErrorToError404(err), "error retrieving DRS job")
+		return diag.Errorf("error creating DRS v3 client, error: %s", err)
 	}
-	detail := detailResp.Results[0]
 
-	if utils.StrSliceContains(
-		[]string{"RELEASE_RESOURCE_COMPLETE", "RELEASE_RESOURCE_STARTED", "RELEASE_RESOURCE_FAILED"}, detail.Status) {
-		return nil
-	}
+	// update name and description
+	if d.HasChanges("name", "description") {
+		detailResp, err := jobs.Get(client, jobs.QueryJobReq{Jobs: []string{d.Id()}})
+		if err != nil {
+			return common.CheckDeletedDiag(d, parseDrsJobErrorToError404(err), "error retrieving DRS job")
+		}
+		detail := detailResp.Results[0]
 
-	updateParams := jobs.UpdateReq{
-		Jobs: []jobs.UpdateJobReq{
-			{
-				JobId:       d.Id(),
-				Name:        d.Get("name").(string),
-				Description: d.Get("description").(string),
+		if utils.StrSliceContains(
+			[]string{"RELEASE_RESOURCE_COMPLETE", "RELEASE_RESOURCE_STARTED", "RELEASE_RESOURCE_FAILED"}, detail.Status) {
+			return nil
+		}
+
+		updateParams := jobs.UpdateReq{
+			Jobs: []jobs.UpdateJobReq{
+				{
+					JobId:       d.Id(),
+					Name:        d.Get("name").(string),
+					Description: d.Get("description").(string),
+				},
 			},
-		},
+		}
+
+		_, err = jobs.Update(client, updateParams)
+		if err != nil {
+			return diag.Errorf("update job: %s failed,error: %s", d.Id(), err)
+		}
 	}
 
-	_, err = jobs.Update(client, updateParams)
-	if err != nil {
-		return diag.Errorf("update job: %s failed,error: %s", d.Id(), err)
+	// update tags
+	if d.HasChange("tags") {
+		tagErr := utils.UpdateResourceTags(clientV5, d, "jobs/"+d.Get("type").(string), d.Id())
+		if tagErr != nil {
+			return diag.Diagnostics{
+				diag.Diagnostic{
+					Severity: diag.Warning,
+					Summary:  fmt.Sprintf("error updating tags for DRS job(%s) with V5 API: %s", d.Id(), tagErr),
```

Comment:
I'd like to write like this:
```suggestion
					Summary:  fmt.Sprintf("failed to update tags for DRS job(%s): %s", d.Id(), tagErr),
```